R/dataframe.R

```diff
@@ -182,24 +182,63 @@ as_data_frame.matrix <- function(x, ...) {
   as_data_frame(as.data.frame(x))
 }
 
-#' Convert row names to an explicit variable.
+#' Conversion between rownames and a column in data frame
+#'
+#' @details add_rownames: Convert row names to an explicit variable.
```

Comment:
Prose would work better here, also a `@description` section is missing. (The caption is repeated, which is not particularly nice.)